### DISCUSSION TITLE
Window Geometry

### DISCUSSION THREAD

**Discussion by js3410** [ImageMagick/ImageMagick#7336] [Help]
### ImageMagick version

unknown

### Operating system, version and so on

Linux

### Description

I am using ImageMagick as a jpg image viewer as it is my only choice on the DoD HPC system I use.  I realize that it is overkill for this purpose.  I am unhappy that the window pops up on the far left of my left monitor and would like to position it more favorably.  I couldn't find a way to do this.  I hope someone can help.

**Comment by urban-warrior** [comment]
Try this command:

```
display -geometry +500+500 myIMage.jpg
```

**Comment by js3410** [comment]
Does this specify position or size?
